TEMPLATES.md

```diff
@@ -111,3 +111,8 @@ before gathering the artifacts.
 `ci-operator.openshift.io/always-show-output`:
 Will output the logs of all the containers in the pod, no matter what the exit code was.
 The value should be `true` to enable this feature.
+
+`ci-operator.openshift.io/always-show-container-output`:
+Will output the logs of a list of containers in the pod, no matter what the exit code was.
+To specify multiple containers the value should be split by `,`. example: `setup,teardown,test`
```

Comment:
This document is lacking a description of how output from containers is handled by default. We should add that to contrast it with the description of the annotation here.